src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -50,5 +73,19 @@ public CacheOptions(bool useSharedCache)
         /// </remarks>
         public bool UseSharedCache { get; set; }
```

Comment:
Should this be settable outside of the constructor?
What happens if this is changed after some time?

Reply:
This would only be read once during app creation.